site/AboutThisData.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from "react"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faArrowDown } from "@fortawesome/free-solid-svg-icons"
+import dayjs from "dayjs"
+
+import {
+    DATAPAGE_ABOUT_THIS_DATA_SECTION_ID,
+    SimpleMarkdownText,
+    ExpandableToggle,
+    HtmlOrSimpleMarkdownText,
+} from "@ourworldindata/components"
+import { DataPageDataV2, OwidOrigin } from "@ourworldindata/types"
+import { uniq } from "@ourworldindata/utils"
+import KeyDataTable from "./KeyDataTable.js"
+
+function getYearSuffixFromOrigin(origin: OwidOrigin) {
+    const year = origin.dateAccessed
+        ? dayjs(origin.dateAccessed, ["YYYY-MM-DD", "YYYY"]).year()
+        : origin.datePublished
+          ? dayjs(origin.datePublished, ["YYYY-MM-DD", "YYYY"]).year()
+          : undefined
+    if (year) return ` (${year})`
+    else return ""
+}
+
+export default function AboutThisData({
+    datapageData,
```

Comment:
It was easy to unwrap this object into top-level props in `MetadataSection`, but here it was more tricky, because we pass the whole `datapageData` to functions deep down that expect an object like `makeLastUpdated` and others in `KeyDataTable`, which are also used in grapher. Let me know if you think it's worth it to unwrap it also here.

In that case, our options are to construct ad-hoc objects to pass to each of those helper functions or also refactor the helper functions to take multiple args, instead of a single object.